public/index.html

```diff
@@ -6,7 +6,7 @@
 
     <title>Eddie Jaoude Community</title>
 
-    <link rel="icon" type="image/png" href="/favicon.png" />
+    <link rel="icon" type="image/ico" href="./favicon.ico" />
```

Comment:
This file does not exist?

Reply:
![image](https://user-images.githubusercontent.com/42907572/95677211-99c05d00-0be1-11eb-8005-878a01480189.png)

Yes sir it is right there. ☝🏻

Reply:
yes, this file is also added on this PR in the public folder 😃

Reply:
Oops sorry I missed it 🤦‍♂️